migrate/data/migrations/1701872471_initialise_schema.down.sql

```diff
@@ -0,0 +1,9 @@
+-- Commented out the statements below as you never want to undo the initialise.
+-- DROP TABLE IF EXISTS users;
```

Comment:
You are correct, it must be deleted, it is a leftover of the migration framework example.

As an example migration I moved the geo_polygon SQL code to a separate script. As long as we are not in production, we could always move it back again to the initialise script. But thought it might be nice as an example of a migration.